src/MenuItem/MenuItemInterface.php

```diff
@@ -19,6 +19,11 @@ public function getRows(MenuStyle $style, bool $selected = false) : array;
      */
     public function getText() : string;
 
+    /**
+     * Set the raw string of text
+     */
+    public function setText(string $text) : void;
```

Comment:
This doesn't need to be on the interface, when you are changing the text you will know what item you are dealing with and therefore will know the methods, no need to change this.